ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/ComposeFormFields/ComposeFormFields.test.tsx

```diff
@@ -165,6 +165,80 @@ describe("ComposeFormFields", () => {
     ).toBe(true);
   });
 
+  it("does not allow hugepage backing non-LXD pods", async () => {
+    const state = { ...initialState };
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <Formik initialValues={{}} onSubmit={jest.fn()}>
+          <ComposeFormFields
+            architectures={[]}
+            available={{
+              cores: 2,
+              hugepages: 0,
```

Comment:
Do you need to add some free huge pages here as you could just be testing the same mechanic twice?

Reply:
I don't think so - this test is just for non-LXD pods so adding any here won't change anything. Plus I think it's not possible to have this number be anything other than 0 for non-LXD pods?